packages/storybook/stories/Examples/TeamsInterop/__snapshots__/Lobby.stories.storyshot

```diff
@@ -43,7 +43,7 @@ exports[`storybook snapshot tests Storyshots Examples/Teams Interop/Lobby Lobby
             style={
               Object {
                 "left": "50%",
-                "marginTop": "2.125rem",
+                "marginTop": "-2.125rem",
```

Comment:
I noticed that this is supposed to be negative to pull the "You're in the lobby" div up to compensate for the ControlBar